specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -1,3215 +1,3193 @@
-			"type": "object",
-			"properties": {
-				"description": {
-					"type": "string"
-				},
-				"filterEnabled": {
-					"type": "boolean"
-				},
-				"groupingEnabled": {
-					"type": "boolean"
-				},
-				"data": {
-					"type": "array",
-					"items": {
-						"type": "string"
-					}
-				},
-				"total": {
-					"format": "int32",
-					"type": "integer"
-				},
-				"category": {
-					"type": "string"
-				},
-				"usageStart": {
-					"format": "date-time",
-					"type": "string"
-				},
-				"usageEnd": {
-					"format": "date-time",
-					"type": "string"
-				},
-				"nextLink": {
-					"type": "string"
-				}
-			}
-		},
-		"QueryResult": {
-			"description": "Result of query. It contains all columns listed under groupings and aggregation.",
-			"type": "object",
-			"properties": {
-				"value": {
-					"description": "The list of usage data.",
-					"type": "array",
-					"readOnly": true,
-					"items": {
-						"$ref": "#/definitions/Query"
-					}
-				}
-			}
-		},
-		"Query": {
-			"type": "object",
-			"allOf": [{
-				"$ref": "#/definitions/Resource"
-			}],
-			"properties": {
-				"properties": {
-					"x-ms-client-flatten": true,
-					"$ref": "#/definitions/QueryProperties",
-					"title": "Query properties"
-				}
-			}
-		},
-		"QueryProperties": {
-			"type": "object",
-			"properties": {
-				"nextLink": {
-					"type": "string"
-				},
-				"columns": {
-					"description": "Array of columns",
-					"type": "array",
-					"items": {
-						"$ref": "#/definitions/QueryColumn"
-					}
-				},
-				"rows": {
-					"type": "array",
-					"items": {
-						"type": "array",
-						"items": {}
-					}
-				}
-			}
-		},
-		"QueryColumn": {
-			"type": "object",
-			"properties": {
-				"name": {
-					"type": "string"
-				},
-				"type": {
-					"type": "string"
-				}
-			}
-		},
-		"ConnectorDefinitionListResult": {
-			"description": "Result of listing connector definitions. It contains a list of available connector definitions in the scope provided.",
-			"properties": {
-				"value": {
-					"description": "The list of connector definitions.",
-					"type": "array",
-					"readOnly": true,
-					"items": {
-						"$ref": "#/definitions/ConnectorDefinition"
-					}
-				}
-			}
-		},
-		"ConnectorDefinition": {
-			"type": "object",
-			"description": "The Connector model definition",
-			"properties": {
-				"kind": {
-					"type": "string",
-					"description": "Connector kind (eg aws)"
-				},
-				"id": {
-					"type": "string",
-					"readOnly": true,
-					"description": "Connector id"
-				},
-				"name": {
-					"type": "string",
-					"readOnly": true,
-					"description": "Connector name"
-				},
-				"type": {
-					"type": "string",
-					"readOnly": true,
-					"description": "Connector type"
-				},
-				"properties": {
-					"x-ms-client-flatten": true,
-					"$ref": "#/definitions/ConnectorProperties",
-					"description": "Connector properties"
-				}
-			},
-			"x-ms-azure-resource": true
-		}
```

Comment:
What does "any of its descendants" mean here? Is there anyway for the customer to bypass RBAC with this API (i.e. fetching data they shouldn't have access to)?

Reply:
With or without the $recurse parameter, this method only returns external subscriptions associated to management groups that they have suitable permissions for. However the $recurse parameter just indicates if the management group mentioned refers to only that MG or its descendants as well.